s3torchconnectorclient/rust/src/mountpoint_s3_client.rs

```diff
@@ -42,26 +44,33 @@ pub struct MountpointS3Client {
     no_sign_request: bool,
     #[pyo3(get)]
     user_agent_prefix: String,
+    #[pyo3(get)]
+    endpoint: String,
 
     owner_pid: Pid,
 }
 
 #[pymethods]
 impl MountpointS3Client {
     #[new]
-    #[pyo3(signature = (region, user_agent_prefix="".to_string(), throughput_target_gbps=10.0, part_size=8*1024*1024, profile=None, no_sign_request=false))]
+    #[pyo3(signature = (region, user_agent_prefix="".to_string(), throughput_target_gbps=10.0, part_size=8*1024*1024, profile=None, no_sign_request=false, endpoint="".to_string()))]
```

Comment:
Can we also change this to be default `None`

Reply:
Changed the default value of endpoint parameter to None instead of empty string